contract/src/execute.rs

```diff
@@ -338,7 +338,8 @@ fn withdraw(
 
     let Sell { funds, returned } = market.sell(outcome, tokens)?;
 
-    if share_info.get_outcome(&market, outcome)?.is_zero() {
+    if share_info.get_outcome(&market, outcome, false)?.is_zero() {
+        println!("here1");
```

Comment:
I believe you didn't mean to check this in..

Reply:
Doh, good catch, thanks!